swaylortift/swaylortift/Source/Util/LogFile.swift

```diff
@@ -44,7 +45,7 @@ public class LogFile
 	///
 	public func append(content: String, encoding: String.Encoding = .utf16) -> Bool
 	{
-		if let fileHandle = FileHandle(forWritingAtPath: filePath)
+		if let fileHandle = fileHandle
```

Comment:
Hey, I found we can't use `FileHandler` in this way. 
It will close the file when deallocated, if you keep it in the property it will never close the file even user kill the app.

Reply:
Thank you for pointing it out. I think I can close the file when deallocating. What do you think?